src/data/files/file.model.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 The Ontario Institute for Cancer Research. All rights reserved
+ * Copyright (c) 2023 The Ontario Institute for Cancer Research. All rights reserved
```

Comment:
Will have a separate PR to update all files with this copyright. All modified files in this PR have this change.

Reply:
this is my approach too, makes it more obvious the file has been modified recently (and potentially improved)